PowerShellEditorServices.sln

```diff
@@ -1,99 +1,99 @@
-﻿
-Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
Check out this StackOverflow answer: http://stackoverflow.com/a/20653073

Maybe core.autocrlf needs to be set to 'input' ?  I've never had to use that before, though